packages/desktop-client/src/components/reports/graphs/StackedBarGraph.tsx

```diff
@@ -83,7 +89,11 @@ const CustomTooltip = ({
                       key={pay.name}
                       left={pay.name}
                       right={amountToCurrency(pay.value)}
-                      style={{ color: pay.color }}
+                      style={{
+                        color: pay.color,
+                        fontWeight: tooltip === pay.name ? 'bold' : 'inherit',
+                        fontSize: tooltip === pay.name ? 15 : 'inherit',
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏it's a bit disorienting for the list to change size when changing hover target. WDYT about changing this to add an underline instead?